### DISCUSSION TITLE
Separate beta and GA apps in app store?

### DISCUSSION THREAD

**Discussion by Splinter7914** [jellyfin/jellyfin-androidtv#3885] [General]
I am aware I can sign up for beta program and then get early access. However that then requires one to delete the app and leave beta if wanting to switch to GA app as future bug fixes do not get any beta releases. 

This also means all settings are lost and need to re set it up.

It could be useful if there was a beta and GA app on the app store which would then allow one to run both beta and ga apps at the same time.

**Comment by nielsvanvelzen** [comment]
There's no plans to add a separate app just for the beta's. You can already sideload debug versions if you want to test upcoming versions while also keeping the "stable" versions installed.

**Reply by Splinter7914** [reply]
Fair point, i should have realized I could side load it. I thought having a specific beta app would make it easier for users to provide feedback on upcoming changes.

Will side load the next beta 👍🏽

**Reply by nielsvanvelzen** [reply]
It would definitely make it easier, but at the cost of extra work for me. So far the beta releases haven't been that successful for me so I don't want to spent that much extra time on it. Perhaps in the future!
